### ISSUE TITLE
Remove Map sprite markers in LESS themes

### ISSUE THREAD

**Issue by Alex-Bubblemaster** [telerik/kendo-ui-core#3368]
### Bug report

Map renders both sprite images and font icons:

![image](https://user-images.githubusercontent.com/10980842/27782202-2ec95d00-5fda-11e7-8419-629def71a579.png)

### Reproduction of the problem

[Official demos](http://demos.telerik.com/kendo-ui/map/index)
Reported in ticket ID 1117151

### Expected/desired behavior

Sprite references should be removed from [LESS themes](https://github.com/telerik/kendo/search?utf8=%E2%9C%93&q=%40%7Bimage-folder%7D%2Fmarkers.png&type=)

### Workaround

```
 <style>
    .k-map .k-marker {
      background-image: none;
    }
  </style>
```
[Dojo](http://dojo.telerik.com/USepa)

### Environment

* **Kendo UI version:** 2017.2.504

**Comment by Alex-Bubblemaster** [comment]
Raising priority - tickets 1128684 and 1128713

**Comment by VicTachev** [comment]
This is fixed in 2017 R3 version (2017.3.913) - [dojo](http://dojo.telerik.com/USepa/7)
